README.md

```diff
@@ -252,6 +252,28 @@ Run it with:
 yarn stats
 ```
 
+### Organization Configuration
+
+Organization configuration can be generated by running:
+`yarn add-org`
+
+Organization configuration is generated from the template present in `internals/generators`.
+Default configuration is present in `internals/config`. If the configuration file of a specific
```

Comment:
I'd specify `/internals/generators/config.yml.hbs` just for the sake of explicitness..